IoT-Projects/ESP8266/Brushless Speed Controller/UBIDOTS_ESP8266_ESC-30A_BRUSHLESS_SPEED_CONTROLLER.ino

```diff
@@ -0,0 +1,68 @@
+/****************************************
+   Include Libraries
+ ****************************************/
+
+#include "UbidotsESPMQTT.h"
+#include <Servo.h>
+/****************************************
+   Define Constants
+ ****************************************/
+#define TOKEN "..........................." // Your Ubidots TOKEN
+#define WIFINAME "........." //Your SSID
+#define WIFIPASS "........." // Your Wifi Pass
+#define DEVICE_LABEL  "motor"  // Put here your Ubidots device label
+#define VARIABLE  "speed"  // Put here your Ubidots variable label
+#define MotorPin D5 //NodeMCU pin where the signal for the ESC comes out
+Servo ESC;
+float value=0; // To store incoming value.
+float MotorSpeed=0;
+
+Ubidots client(TOKEN);
+
+/****************************************
+   Auxiliar Functions
+ ****************************************/
+ 
+// cast from an array of chars to float value.
+float btof(byte * payload, unsigned int length) {
+    char * demo_ = (char *) malloc(sizeof(char) * 10);
```

Comment:
2 spaces instead of 4 spaces, please follow the style guide from google: https://google.github.io/styleguide/cppguide.html#Functions